css-canvas-medical/index.html

```diff
@@ -0,0 +1,137 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <meta http-equiv="X-UA-Compatible" content="ie=edge" />
+    <title>CSS Challenge - Canvas Medical</title>
+    <link rel="stylesheet" href="fonts.css" />
+    <link rel="stylesheet" href="reset.css" />
+    <link rel="stylesheet" href="styles.css" />
+  </head>
+  <body>
+    <header class="flex-row justify-end position-relative">
+      <div class="canvas-blue">
+        <a href="index.html">
+          <img src="images/logo-blue.svg" alt="canvas medical logo" />
+        </a>
+      </div>
+      <div class="header-link-container light-grey-link-container">
+        <a class="light-grey-link" href="product.html">Product</a>
+        <a class="light-grey-link" href="implementation.html">Implementation</a>
+        <a class="light-grey-link" href="about-us.html">About Us</a>
+        <a class="light-grey-link" href="blog.html">Blog</a>
+        <a
+          class="header-link blue-button contact-us l-link"
+          href="contact-us.html"
+          >Contact Us</a
+        >
+      </div>
```

Comment:
Because this is technically a _list_ of links, I would have made this a `nav` element with a `ul` and then each `a` be inside of an `li`.
```
<nav class="column-three-quarter justify-flex-end">
  <ul class="nav-links">
    <li><a href="#">Product</a></li>
    <li><a href="#">Implementation</a></li>
    <li><a href="#">About Us</a></li>
    <li><a href="#">Blog</a></li>
    <li class="action"><a href="#">Contact Us</a></li>
  </ul>
</nav>
```